stingray/pulse/tests/test_modeling.py

```diff
@@ -0,0 +1,84 @@
+import numpy as np
+from stingray.pulse.modeling import fit_sinc, fit_gaussian
+
+np.random.seed(0)
+
```

Comment:
I like the tests, but as far as I can see (I might have missed them), there are currently no tests for the `fit_gaussian` and `fit_sinc` functions. I think it would be useful to have them covered by tests, too. At least some checks about getting the input/output right, and maybe with a simulated light curve and simple checks that the inferred parameters are reasonably close-ish to those put in/